go/private/sdk.bzl

```diff
@@ -545,7 +545,8 @@ def _detect_sdk_version(ctx, goroot):
     version_file_path = goroot + "/VERSION"
     if ctx.path(version_file_path).exists:
         # VERSION file has version prefixed by go, eg. go1.18.3
-        version = ctx.read(version_file_path)[2:]
+        # 1.21: The version is the first line
+        version = ctx.read(version_file_path)[2:].splitlines()[0]
```

Comment:
Could you swap this so that we split lines before we take the substring? Both are equivalent for valid VERSION files, but I find this way more natural and less confusing if the format isn't valid.

Reply:
Good suggestion. I took the liberty of splitting this into two lines with a new intermediate variable. I think this makes it clearer that there are two operations here: take the first line, then strip off the leading "go"